docs/design.md

```diff
@@ -1,24 +1,25 @@
 # Design
 
-Trino Gateway is composed of the following main components:
```

Comment:
Getting rid of this is good imho .. but we need some more high level design info as well. We can add later though.